stories/nldas3.stories.mdx

```diff
@@ -9,7 +9,7 @@ media:
   author:
     name: LDAS-NASA
     url: https://eoimages.gsfc.nasa.gov/images/imagerecords/151000/151897/mississippi_oli_2023259_lrg.jpg
-pubDate: 2024-07-25
+pubDate: 2024-10-22
```

Comment:
Does this date need updated to `2024-10-23`?

Reply:
yes we need an recent date